beebop/app.py

```diff
@@ -164,5 +212,55 @@ def get_result_internal(hash, redis):
         return jsonify(response_failure("Unknown project hash"))
 
 
+def send_zip_internal(project_hash, type, cluster, storage_location):
+    fs = PoppunkFileStore(storage_location)
+    if type == 'microreact':
+        path_folder = fs.output_microreact(project_hash, cluster)
+    elif type == 'network':
+        path_folder = fs.output_network(project_hash)
+    # generate zipfile
+    memory_file = generateZip(path_folder)
+    return send_file(memory_file,
+                     download_name=type + '.zip',
+                     as_attachment=True)
+
+
+def generate_microreact_url_internal(microreact_api_new_url,
+                                     project_hash,
+                                     cluster,
+                                     api_token,
+                                     storage_location):
+    fs = PoppunkFileStore(storage_location)
+
+    data = {}
+    path_csv = fs.microreact_csv(project_hash, cluster)
+    add_data(path_csv, 'csv', data)
+    path_dot = fs.microreact_dot(project_hash, cluster)
+    add_data(path_dot, 'dot', data)
+    # nwk is only available where cluster has >=3 samples
+    path_nwk = fs.microreact_nwk(project_hash, cluster)
+    if os.path.exists(path_nwk):
```

Comment:
I am a bit confused about what is going on here. Area the files in resources here templates that we're filling in?

Reply:
Yes. I followed [John's blog post](http://www.johnlees.me/blog/2022/05/06/using-the-new-microreact-api/) on the new microreact API, that suggests doing it that way. 
I tried using the conversion endpoint, which should take the old API request payload (which was the tree, network and csv data) and generate a new .microreact file which is required for the new API, but was not successful in doing so. Unfortunately I could not find proper documentation on this ( especially on how the old payload should exactly look like), and also John mentioned there are issues when using a network plot, so gave up on this.